### ISSUE TITLE
 error occurred when running the  c3d_sport1m_feature_extraction_video.sh

### ISSUE THREAD

**Issue by jijilulu** [facebook/C3D#308]
zhao@104:~/Action Recon/C3D-master/C3D-v1.0/examples/c3d_feature_extraction$ sh c3d_sport1m_feature_extraction_video.sh
WARNING: Logging before InitGoogleLogging() is written to STDERR
E0907 11:31:45.233443 29887 extract_image_features.cpp:64] Using GPU #0
I0907 11:31:45.234361 29887 net.cpp:76] Creating Layer data
I0907 11:31:45.234386 29887 net.cpp:112] data -> data
I0907 11:31:45.234395 29887 net.cpp:112] data -> label
I0907 11:31:45.234405 29887 video_data_layer.cpp:283] Opening file prototxt/input_list_video.txt
I0907 11:31:45.234441 29887 video_data_layer.cpp:317] A total of 16 video chunks.
I0907 11:31:45.324717 29887 video_data_layer.cpp:365] output data size: 50,3,16,112,112
I0907 11:31:45.324754 29887 video_data_layer.cpp:387] Loading mean file from sport1m_train16_128_mean.binaryproto
I0907 11:31:45.338290 29887 net.cpp:127] Top shape: 50 3 16 112 112 (30105600)
I0907 11:31:45.338316 29887 net.cpp:127] Top shape: 50 1 1 1 1 (50)
I0907 11:31:45.338322 29887 net.cpp:159] data does not need backward computation.
I0907 11:31:45.338333 29887 net.cpp:76] Creating Layer conv1a
I0907 11:31:45.338338 29887 net.cpp:86] conv1a <- data
I0907 11:31:45.338346 29887 net.cpp:112] conv1a -> conv1a
I0907 11:31:45.338587 29887 net.cpp:127] Top shape: 50 64 16 112 112 (642252800)
I0907 11:31:45.338595 29887 net.cpp:154] conv1a needs backward computation.
I0907 11:31:45.338601 29887 net.cpp:76] Creating Layer relu1a
I0907 11:31:45.338605 29887 net.cpp:86] relu1a <- conv1a
I0907 11:31:45.338610 29887 net.cpp:100] relu1a -> conv1a (in-place)
I0907 11:31:45.338615 29887 net.cpp:127] Top shape: 50 64 16 112 112 (642252800)
I0907 11:31:45.338619 29887 net.cpp:154] relu1a needs backward computation.
I0907 11:31:45.338625 29887 net.cpp:76] Creating Layer pool1
I0907 11:31:45.338629 29887 net.cpp:86] pool1 <- conv1a
I0907 11:31:45.338634 29887 net.cpp:112] pool1 -> pool1
I0907 11:31:45.338644 29887 net.cpp:127] Top shape: 50 64 16 56 56 (160563200)
I0907 11:31:45.338649 29887 net.cpp:154] pool1 needs backward computation.
I0907 11:31:45.338654 29887 net.cpp:76] Creating Layer conv2a
I0907 11:31:45.338659 29887 net.cpp:86] conv2a <- pool1
I0907 11:31:45.338663 29887 net.cpp:112] conv2a -> conv2a
I0907 11:31:45.343490 29887 net.cpp:127] Top shape: 50 128 16 56 56 (321126400)
I0907 11:31:45.343510 29887 net.cpp:154] conv2a needs backward computation.
I0907 11:31:45.343518 29887 net.cpp:76] Creating Layer relu2a
I0907 11:31:45.343523 29887 net.cpp:86] relu2a <- conv2a
I0907 11:31:45.343528 29887 net.cpp:100] relu2a -> conv2a (in-place)
I0907 11:31:45.343533 29887 net.cpp:127] Top shape: 50 128 16 56 56 (321126400)
I0907 11:31:45.343536 29887 net.cpp:154] relu2a needs backward computation.
I0907 11:31:45.343542 29887 net.cpp:76] Creating Layer pool2
I0907 11:31:45.343546 29887 net.cpp:86] pool2 <- conv2a
I0907 11:31:45.343551 29887 net.cpp:112] pool2 -> pool2
I0907 11:31:45.343556 29887 net.cpp:127] Top shape: 50 128 8 28 28 (40140800)
I0907 11:31:45.343561 29887 net.cpp:154] pool2 needs backward computation.
I0907 11:31:45.343570 29887 net.cpp:76] Creating Layer conv3a
I0907 11:31:45.343577 29887 net.cpp:86] conv3a <- pool2
I0907 11:31:45.343585 29887 net.cpp:112] conv3a -> conv3a
I0907 11:31:45.362341 29887 net.cpp:127] Top shape: 50 256 8 28 28 (80281600)
I0907 11:31:45.362366 29887 net.cpp:154] conv3a needs backward computation.
I0907 11:31:45.362375 29887 net.cpp:76] Creating Layer relu3a
I0907 11:31:45.362380 29887 net.cpp:86] relu3a <- conv3a
I0907 11:31:45.362386 29887 net.cpp:100] relu3a -> conv3a (in-place)
I0907 11:31:45.362391 29887 net.cpp:127] Top shape: 50 256 8 28 28 (80281600)
I0907 11:31:45.362396 29887 net.cpp:154] relu3a needs backward computation.
I0907 11:31:45.362401 29887 net.cpp:76] Creating Layer conv3b
I0907 11:31:45.362406 29887 net.cpp:86] conv3b <- conv3a
I0907 11:31:45.362411 29887 net.cpp:112] conv3b -> conv3b
I0907 11:31:45.399545 29887 net.cpp:127] Top shape: 50 256 8 28 28 (80281600)
I0907 11:31:45.399567 29887 net.cpp:154] conv3b needs backward computation.
I0907 11:31:45.399574 29887 net.cpp:76] Creating Layer relu3b
I0907 11:31:45.399580 29887 net.cpp:86] relu3b <- conv3b
I0907 11:31:45.399585 29887 net.cpp:100] relu3b -> conv3b (in-place)
I0907 11:31:45.399590 29887 net.cpp:127] Top shape: 50 256 8 28 28 (80281600)
I0907 11:31:45.399595 29887 net.cpp:154] relu3b needs backward computation.
I0907 11:31:45.399600 29887 net.cpp:76] Creating Layer pool3
I0907 11:31:45.399605 29887 net.cpp:86] pool3 <- conv3b
I0907 11:31:45.399610 29887 net.cpp:112] pool3 -> pool3
I0907 11:31:45.399616 29887 net.cpp:127] Top shape: 50 256 4 14 14 (10035200)
I0907 11:31:45.399621 29887 net.cpp:154] pool3 needs backward computation.
I0907 11:31:45.399626 29887 net.cpp:76] Creating Layer conv4a
I0907 11:31:45.399631 29887 net.cpp:86] conv4a <- pool3
I0907 11:31:45.399641 29887 net.cpp:112] conv4a -> conv4a
I0907 11:31:45.473834 29887 net.cpp:127] Top shape: 50 512 4 14 14 (20070400)
I0907 11:31:45.473853 29887 net.cpp:154] conv4a needs backward computation.
I0907 11:31:45.473860 29887 net.cpp:76] Creating Layer relu4a
I0907 11:31:45.473866 29887 net.cpp:86] relu4a <- conv4a
I0907 11:31:45.473871 29887 net.cpp:100] relu4a -> conv4a (in-place)
I0907 11:31:45.473876 29887 net.cpp:127] Top shape: 50 512 4 14 14 (20070400)
I0907 11:31:45.473881 29887 net.cpp:154] relu4a needs backward computation.
I0907 11:31:45.473887 29887 net.cpp:76] Creating Layer conv4b
I0907 11:31:45.473891 29887 net.cpp:86] conv4b <- conv4a
I0907 11:31:45.473896 29887 net.cpp:112] conv4b -> conv4b
I0907 11:31:45.624202 29887 net.cpp:127] Top shape: 50 512 4 14 14 (20070400)
I0907 11:31:45.624227 29887 net.cpp:154] conv4b needs backward computation.
I0907 11:31:45.624234 29887 net.cpp:76] Creating Layer relu4b
I0907 11:31:45.624240 29887 net.cpp:86] relu4b <- conv4b
I0907 11:31:45.624246 29887 net.cpp:100] relu4b -> conv4b (in-place)
I0907 11:31:45.624251 29887 net.cpp:127] Top shape: 50 512 4 14 14 (20070400)
I0907 11:31:45.624256 29887 net.cpp:154] relu4b needs backward computation.
I0907 11:31:45.624263 29887 net.cpp:76] Creating Layer pool4
I0907 11:31:45.624266 29887 net.cpp:86] pool4 <- conv4b
I0907 11:31:45.624271 29887 net.cpp:112] pool4 -> pool4
I0907 11:31:45.624277 29887 net.cpp:127] Top shape: 50 512 2 7 7 (2508800)
I0907 11:31:45.624281 29887 net.cpp:154] pool4 needs backward computation.
I0907 11:31:45.624287 29887 net.cpp:76] Creating Layer conv5a
I0907 11:31:45.624291 29887 net.cpp:86] conv5a <- pool4
I0907 11:31:45.624296 29887 net.cpp:112] conv5a -> conv5a
I0907 11:31:45.775822 29887 net.cpp:127] Top shape: 50 512 2 7 7 (2508800)
I0907 11:31:45.775845 29887 net.cpp:154] conv5a needs backward computation.
I0907 11:31:45.775852 29887 net.cpp:76] Creating Layer relu5a
I0907 11:31:45.775857 29887 net.cpp:86] relu5a <- conv5a
I0907 11:31:45.775863 29887 net.cpp:100] relu5a -> conv5a (in-place)
I0907 11:31:45.775868 29887 net.cpp:127] Top shape: 50 512 2 7 7 (2508800)
I0907 11:31:45.775872 29887 net.cpp:154] relu5a needs backward computation.
I0907 11:31:45.775878 29887 net.cpp:76] Creating Layer conv5b
I0907 11:31:45.775883 29887 net.cpp:86] conv5b <- conv5a
I0907 11:31:45.775887 29887 net.cpp:112] conv5b -> conv5b
I0907 11:31:45.924778 29887 net.cpp:127] Top shape: 50 512 2 7 7 (2508800)
I0907 11:31:45.924803 29887 net.cpp:154] conv5b needs backward computation.
I0907 11:31:45.924810 29887 net.cpp:76] Creating Layer relu5b
I0907 11:31:45.924816 29887 net.cpp:86] relu5b <- conv5b
I0907 11:31:45.924821 29887 net.cpp:100] relu5b -> conv5b (in-place)
I0907 11:31:45.924827 29887 net.cpp:127] Top shape: 50 512 2 7 7 (2508800)
I0907 11:31:45.924831 29887 net.cpp:154] relu5b needs backward computation.
I0907 11:31:45.924837 29887 net.cpp:76] Creating Layer pool5
I0907 11:31:45.924841 29887 net.cpp:86] pool5 <- conv5b
I0907 11:31:45.924845 29887 net.cpp:112] pool5 -> pool5
I0907 11:31:45.924851 29887 net.cpp:127] Top shape: 50 512 1 4 4 (409600)
I0907 11:31:45.924855 29887 net.cpp:154] pool5 needs backward computation.
I0907 11:31:45.924861 29887 net.cpp:76] Creating Layer fc6-1
I0907 11:31:45.924865 29887 net.cpp:86] fc6-1 <- pool5
I0907 11:31:45.924870 29887 net.cpp:112] fc6-1 -> fc6-1
I0907 11:31:46.613437 29887 net.cpp:127] Top shape: 50 4096 1 1 1 (204800)
I0907 11:31:46.613473 29887 net.cpp:154] fc6-1 needs backward computation.
I0907 11:31:46.613481 29887 net.cpp:76] Creating Layer relu6
I0907 11:31:46.613487 29887 net.cpp:86] relu6 <- fc6-1
I0907 11:31:46.613492 29887 net.cpp:100] relu6 -> fc6-1 (in-place)
I0907 11:31:46.613498 29887 net.cpp:127] Top shape: 50 4096 1 1 1 (204800)
I0907 11:31:46.613502 29887 net.cpp:154] relu6 needs backward computation.
I0907 11:31:46.613507 29887 net.cpp:76] Creating Layer drop6
I0907 11:31:46.613512 29887 net.cpp:86] drop6 <- fc6-1
I0907 11:31:46.613517 29887 net.cpp:100] drop6 -> fc6-1 (in-place)
I0907 11:31:46.613520 29887 net.cpp:127] Top shape: 50 4096 1 1 1 (204800)
I0907 11:31:46.613525 29887 net.cpp:154] drop6 needs backward computation.
I0907 11:31:46.613530 29887 net.cpp:76] Creating Layer fc7-1
I0907 11:31:46.613535 29887 net.cpp:86] fc7-1 <- fc6-1
I0907 11:31:46.613539 29887 net.cpp:112] fc7-1 -> fc7-1
I0907 11:31:46.946588 29887 net.cpp:127] Top shape: 50 4096 1 1 1 (204800)
I0907 11:31:46.946619 29887 net.cpp:154] fc7-1 needs backward computation.
I0907 11:31:46.946626 29887 net.cpp:76] Creating Layer relu7
I0907 11:31:46.946632 29887 net.cpp:86] relu7 <- fc7-1
I0907 11:31:46.946637 29887 net.cpp:100] relu7 -> fc7-1 (in-place)
I0907 11:31:46.946642 29887 net.cpp:127] Top shape: 50 4096 1 1 1 (204800)
I0907 11:31:46.946646 29887 net.cpp:154] relu7 needs backward computation.
I0907 11:31:46.946652 29887 net.cpp:76] Creating Layer drop7
I0907 11:31:46.946656 29887 net.cpp:86] drop7 <- fc7-1
I0907 11:31:46.946660 29887 net.cpp:100] drop7 -> fc7-1 (in-place)
I0907 11:31:46.946666 29887 net.cpp:127] Top shape: 50 4096 1 1 1 (204800)
I0907 11:31:46.946669 29887 net.cpp:154] drop7 needs backward computation.
I0907 11:31:46.946674 29887 net.cpp:76] Creating Layer fc8-1
I0907 11:31:46.946678 29887 net.cpp:86] fc8-1 <- fc7-1
I0907 11:31:46.946683 29887 net.cpp:112] fc8-1 -> fc8-1
I0907 11:31:46.986932 29887 net.cpp:127] Top shape: 50 487 1 1 1 (24350)
I0907 11:31:46.986948 29887 net.cpp:154] fc8-1 needs backward computation.
I0907 11:31:46.986968 29887 net.cpp:76] Creating Layer prob
I0907 11:31:46.986974 29887 net.cpp:86] prob <- fc8-1
I0907 11:31:46.986979 29887 net.cpp:112] prob -> prob
I0907 11:31:46.986985 29887 net.cpp:127] Top shape: 50 487 1 1 1 (24350)
I0907 11:31:46.986990 29887 net.cpp:154] prob needs backward computation.
I0907 11:31:46.986994 29887 net.cpp:76] Creating Layer accuracy
I0907 11:31:46.986999 29887 net.cpp:86] accuracy <- prob
I0907 11:31:46.987004 29887 net.cpp:86] accuracy <- label
I0907 11:31:46.987009 29887 net.cpp:112] accuracy -> accuracy
I0907 11:31:46.987018 29887 net.cpp:127] Top shape: 1 2 1 1 1 (2)
I0907 11:31:46.987027 29887 net.cpp:154] accuracy needs backward computation.
I0907 11:31:46.987031 29887 net.cpp:165] This network produces output accuracy
I0907 11:31:46.987046 29887 net.cpp:183] Collecting Learning Rate and Weight Decay.
I0907 11:31:46.987056 29887 net.cpp:176] Network initialization done.
I0907 11:31:46.987061 29887 net.cpp:177] Memory required for Data 5653289408
E0907 11:31:47.304532 29887 extract_image_features.cpp:81] Extracting features for 1 batches
F0907 11:31:47.617091 29887 relu_layer.cu:29] # _**Check failed: error == cudaSuccess (11 vs. 0)  invalid argument**_
*** Check failure stack trace: ***
Aborted (core dumped)
Make runtest is already successful,but error occurred when running the  c3d_sport1m_feature_extraction_video.sh.my cuda version is 8.0,and successfully install it.Can you solve my problem?Thanks!

**Comment by amznero** [comment]
You can modify your Makefile.config file.
Change Line 10 ~ 15 like this: 
CUDA_ARCH := -gencode arch=compute_20,code=sm_20 \
		-gencode arch=compute_20,code=sm_21 \
		-gencode arch=compute_30,code=sm_30 \
		-gencode arch=compute_35,code=sm_35 \
		-gencode arch=compute_50,code=sm_50 \
		**-gencode arch=compute_52,code=sm_52 \
		-gencode arch=compute_60,code=sm_60 \
		-gencode arch=compute_61,code=sm_61 \
		-gencode arch=compute_61,code=compute_61**

**Comment by dutran** [comment]
amznero's solution seems right to me. As 1.0 does not have config for CUDA 8.0.
